### PR TITLE
star rating!

### PR DESCRIPTION
**Pull request by lek7518**
finished rating function and helper function

### CODE REVIEW

File: .idea/libraries/jsch_0_1_55.xml
Comment (jak9527) [inline .idea/libraries/jsch_0_1_55.xml:1]: Is it possible for us to remove this change, the other deletion, and the iml change from this pull request? We definitely don't want to change this for everyone

File: src/Application.java [hunk 1]
```diff
@@ -87,6 +87,14 @@ private boolean commandParser(String cmd){
                 searchFriends(cmdArgs.get(1));
             }
         }
+        if(cmd.equals("rate_game")){
+            if(cmdArgs.size() != 3){
+                System.out.println("Usage: rate_game <video game title> <star rating: 1-5>");
+            } else {
+                int rating = Integer.parseInt(cmdArgs.get(2));
```
Comment (jak9527) [inline src/Application.java:94]: check to make sure parseint doesn't return NaN (as in make sure the second argument was properly a number)

File: src/Application.java [hunk 2]
```diff
@@ -148,4 +156,45 @@ private void printResultSet(ResultSet res) throws SQLException{
             System.out.println();
         }
     }
+
+    private int getIdFromTitle(String title){
+        int vg_id;
+        try{
+            Statement st = this.conn.createStatement();
+            ResultSet res = st.executeQuery("select vg_id from video_games where title like %" + title + "%");
+            vg_id = res.getInt("vg_id");
```
Comment (jak9527) [inline src/Application.java:165]: I didn't realize you could do this by col label thats so cool!

File: src/Application.java [hunk 3]
```diff
@@ -148,4 +156,45 @@ private void printResultSet(ResultSet res) throws SQLException{
             System.out.println();
         }
     }
+
```
Comment (jak9527) [inline src/Application.java:159]: All the sql will have to be modified with the whole \" and ' thing to make it work. Other than that it looks good to me.

**Comment by jak9527** [comment]
I am gonna hold off on approving this until we manage to get rid of the deletions and iml changes from the commit, and until the sql is fixed. Check my other comments for details on what I mean.

**Comment by lek7518** [comment]
resolved in a different pull request

**Pull request closed without merging**